src/containers/shared/components/Transaction/EscrowCreate/parser.ts

```diff
@@ -0,0 +1,17 @@
+import { unix } from 'moment'
+import formatAmount from '../../../../../rippled/lib/txSummary/formatAmount'
+
+const EPOCH_OFFSET = 946684800
```

Comment:
You should probably import RIPPLE_EPOCH like in `Description.tsx` instead if that's the same number.

Reply:
There is a dependency cycle error if I import it from rippled

Reply:
I solved this in SignerListSet by moving the function outside of utils.js. That should eliminate the cycle.

Reply:
new dependency errors.... 
https://github.com/ripple/explorer/actions/runs/3064315465/jobs/4947301609